packages/j/jsgradient.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "JSGradient",
```

```suggestion
  "name": "jsgradient",
```
This should match the filename being added, case-sensitive